src/app/src/pages/Profile.tsx

```diff
@@ -152,6 +160,35 @@ export default function Profile({ currentUser }: Props): JSX.Element {
     fetchData();
   }, [author?.id]);
 
+  // Get Current User's followers and update isFollwing is the author is being followed by the current user
+  const [currentUserFollowers, setCurrentUserFollowers] = useState<Author[]>(
+    []
+  );
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const res = api.authors.withId("" + currentUser?.id).followings.list();
+        res.then((followers) => {
+          setCurrentUserFollowers(followers);
+        });
+        let isFollowing = false;
+        if (currentUserFollowers) {
+          for (let i = 0; i < currentUserFollowers.length; i++) {
+            if (currentUserFollowers[i].id === author?.id) {
+              isFollowing = true;
+              break;
+            }
+          }
+
+          setFollowing(isFollowing);
+        }
+      } catch (err) {
+        console.log(err);
+      }
+    }
+    fetchData();
+  }, [author?.id, currentUser?.id, currentUserFollowers]);
+
```

Comment:
It might be easier to just check if the user is following using `api.authors.withId(authorId).followers.isAFollower(followerId)`

Reply:
Though for some reason the backend it is returning it in an object like `{result: boolean}` instead of just a `boolean`. So you'll have to change the type of that function in api.ts from `boolean` to `{result: boolean}`.

Reply:
done